cli/README.md

```diff
@@ -0,0 +1,30 @@
+# Intro to the Command Line/Terminal
```

Comment:
On this row https://github.com/gdisf/teaching-materials/pull/525#issuecomment-320529609
![image](https://user-images.githubusercontent.com/603887/29043580-96d719f8-7b70-11e7-8894-9e5b8c75ccf4.png)